### ISSUE TITLE
JAWS doesn't announce collapsed state of the menu button

### ISSUE THREAD

**Issue by Leuvarden** [FreedomScientific/VFO-standards-support#358]
### Summary

> JAWS doesn't announce expanded/collapsed form buttons with aria-expanded and aria-haspopup=true
> 
> Example:
> 1. With JAWS on, go to https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-2/menubar-2.html 
> 2. Use Left and Right arrows to navigate through the menu buttons in the menubar. JAWS announces menu buttons as "sub menu" and does not announces its collapsed/expanded state

### Expected result

> According to JAWS Aria support, aria-expanded is read by JAWS (https://freedomscientific.github.io/VFO-standards-support/aria.html) Although when aria-expanded is added to the element with role=button and aria-haspopup=true, expanded/collapsed state is not announced. 

#### JAWS version and build number

JAWS 18.0.5038 and JAWS 2018.1811.30
 
#### Operating System and version

Windows 10 Enterprise

#### Browser and version:

Chrome 79.0, Firefox 72.0

**Comment by JAWS-test** [comment]
The problem also occurs with JAWS 2019 and JAWS 2020 in Chrome and Firefox. Here, the status "expanded" and "collapsed" is only output when opening and closing submenus, even if they are automatically closed when navigating through the main menu items. This leads for example to misleading output when navigating from "Text Align" to "Size": ""

**Comment by frex65** [comment]
The test case example doesn't demonstrate the issue that's being described, since each of the elements in its top menubar is a `<a>` element with a `menuitem` role, rather than a button as specified in the issue. The `menuitem` role is not one of the seven roles to which the `aria-expanded` state can be applied. Please can you provide an example that illustrates the problem?

**Comment by JAWS-test** [comment]
> The menuitem role is not one of the seven roles to which the aria-expanded state can be applied. 

aria-expanded is allowed for menuitem: https://www.w3.org/TR/wai-aria-1.2/#menuitem

**Comment by stevefaulkner** [comment]
@JAWS-test you are correct that ARIA 1.2 allows `aria-expanded` on `menuitem`, but it is not yet a W3C recommendation, [ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#menuitem) is a W3C recommendation, in which the `menuitem` role does not allow `aria-expanded`.
